src/matchers/toBeArray/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`.not.toBeArray fails when given an array 1`] = `
+"<dim>expect(</><red>received</><dim>).not.toBeArray(</><dim>)</>
+
+Expected value to not be a string received:
```

Comment:
I think the tests are failing because of these snapshots.

If you run the tests with `yarn test:watch` you should see that the snapshots are different. You can then update it by pressing `u` while watching the tests.

You can then check the snapshot it should say `Expected value to not be an array received:` instead of `Expected value to not be a string received:`